### ISSUE TITLE
Documentation for aic, aicc, bic, adjr2 and r2

### ISSUE THREAD

**Issue by diegozea** [JuliaStats/StatsBase.jl#183]
Hi! That functions aren't documented in the documentation page. Would It be great to have them there. Best,

**Comment by nalimilan** [comment]
Good catch. I think you need to add them to `docs/source/statmodels.rst`. Could you make a PR?

**Comment by nalimilan** [comment]
Note other functions are missing, including `df`, `df_residual`, `nulldeviance` and `nullloglikelihood`. Would be nice to check the list of functions in `statmodel.jl` to ensure we don't forget one.

**Comment by nalimilan** [comment]
Finally, some functions are incorrectly reported as being limited to `RegressionModel` (like `nobs`). `fitted` is also missing from that section.
